command/agentproxyshared/auth/auth_test.go

```diff
@@ -60,7 +60,6 @@ func (u *userpassTestMethod) Shutdown() {
 func TestAuthHandler(t *testing.T) {
 	logger := logging.NewVaultLogger(hclog.Trace)
```

Comment:
Should we use the `TestLogger` for `NewAuthHandler` below, or do we want to stick with the `NewVaultLogger`

Reply:
I chose the latter.